packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -143,9 +143,22 @@ class _VectorGraphicsWidgetState extends State<VectorGraphic> {
     if (pictureInfo == null) {
       return SizedBox(width: widget.width, height: widget.height);
     }
+
+    double? width = widget.width;
```

Comment:
So, if there is no width, height provided, then fall back to the picture's width height, but if one is provided then maintain the aspect ratio?

Could use a short comment.

Reply:
Done